src/stories/components/Card/styles.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Card
+
+.card
+{
+    border-radius: 5px;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    border-bottom-style: solid;
+    border-color: $teal-dark;
+}
+
+.card__container
+{
+    padding: $spacing-3;
+}
+
+.card__details {
+    h3 {
+        margin: 0px;
+    }
+    p {
+        font-family: $font-size-md;;
+    }
+}
+
+.card__extra {
+    opacity: 0.7;
+    margin-bottom: $spacing-3;
+    font-size: $font-size-sm;
+    p {
+        margin: 0px
+    }
+}
+
+.card--active
+{
+    &:hover,
+    &:focus,
+    &:active {
+        border-color: $teal;
+    }
+}
+
+.card--inactive
+{
+    background-color: $gray-1;
+    border-color: black;
+
+    &:hover,
+    &:focus,
+    &:active {
+        border-color: black;
+    }
+
+    .card__button {
+        &:hover,
+        &:focus,
+        &:active,
+        &:visited {
+            background-color: $black;
+            border-radius:0;
+            border-bottom-right-radius: 50px;
+            border-top-left-radius: 50px;
```

Comment:
I believe we can use $spacing-5 for both line 64 and 65.

Reply:
This css is written to prevent the changing of the button. I think this should stay as `50px` to be consistent with the button's css, which is also set at `50px`

Reply:
I see, looks good to me!